src/application/app.test.js

```diff
@@ -11,15 +11,15 @@ describe('a doodle like app', () => {
 
   describe('create event scheduling poll', () => {
     const aTitle = 'Event title'
-    const aDate = new Date()
-    const anEventPollId = app.newEventPoll(aTitle, aDate)
+    const possibleDates = [faker.date.future(), faker.date.future()]
```

Comment:
Is there any reason for having 2 instead of only one possible date? Also, IMO use an external library to generate dates a little bit overkill